tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/demo/tf-interactive-inference-multi-demo.html

```diff
@@ -0,0 +1,273 @@
+    </tf-interactive-inference-dashboard>
+  </template>
+  <script>
+    Polymer({
+      is: "tf-interactive-inference-demo",
+      properties: {
+        model: Object,
+        data: Array,
+        indicesToInfer: {type: Object, value: () => ({})},
+        means: Object,
+        categories: Object,
+        labelVocab: {type: Array, value: ['<=50k', '>50k']},
+      },
+      ready: async function() {
+        this.$.dash.settingsClicked_();
+        this.means = {
+          'age': 38.64358543876172,
+          'education-num': 10.078088530363212,
+          'capital-gain': 1079.0676262233324,
+          'capital-loss': 87.50231358257237,
+          'hours-per-week': 40.422382375824085,
+        }
+        this.categories = {};
+        this.categories['workclass'] = ["Private", "Self-emp-not-inc",
+          "Self-emp-inc", "Federal-gov", "Local-gov", "State-gov",
+          "Without-pay", "Never-worked"];
+        this.categories['education'] = ["Bachelors", "Some-college", "11th",
+          "HS-grad", "Prof-school", "Assoc-acdm", "Assoc-voc", "9th", "7th-8th",
+          "12th", "Masters", "1st-4th", "10th", "Doctorate", "5th-6th",
+          "Preschool"];
+        this.categories['marital-status'] = ["Married-civ-spouse", "Divorced",
+          "Never-married", "Separated", "Widowed", "Married-spouse-absent",
+          "Married-AF-spouse"];
+        this.categories['occupation'] = ["Tech-support", "Craft-repair",
+          "Other-service", "Sales", "Exec-managerial", "Prof-specialty",
+          "Handlers-cleaners", "Machine-op-inspct", "Adm-clerical",
+          "Farming-fishing", "Transport-moving", "Priv-house-serv",
+          "Protective-serv", "Armed-Forces"];
+        this.categories['relationship'] = ["Wife", "Own-child", "Husband",
+          "Not-in-family", "Other-relative", "Unmarried"];
+        this.categories['race'] = ["White", "Asian-Pac-Islander",
+          "Amer-Indian-Eskimo", "Other", "Black"];
+        this.categories['sex'] = ["Female", "Male"];
+        this.categories['native-country'] = ["United-States", "Cambodia",
+          "England", "Puerto-Rico", "Canada", "Germany",
+          "Outlying-US(Guam-USVI-etc)", "India", "Japan", "Greece", "South",
+          "China", "Cuba", "Iran", "Honduras", "Philippines", "Italy", "Poland",
+          "Jamaica", "Vietnam", "Mexico", "Portugal", "Ireland", "France",
+          "Dominican-Republic", "Laos", "Ecuador", "Taiwan", "Haiti",
+          "Columbia", "Hungary", "Guatemala", "Nicaragua", "Scotland",
+          "Thailand", "Yugoslavia", "El-Salvador", "Trinadad&Tobago", "Peru",
+          "Hong", "Holand-Netherlands"];
+
+        this.model = await tf.loadModel(tf.io.browserHTTPRequest(
+          'data/uci/model.json', {credentials: 'include'}));
+        this.model2 = await tf.loadModel(tf.io.browserHTTPRequest(
+          'data/uci/model.json', {credentials: 'include'}));
```

Comment:
if just using 1 model and tweaking the results for simulating a second model, remove the model2 and model2 predict and just copy the res array from the 1 model's predictions and edit them.

Reply:
added a second real model.